core/trino-main/src/test/java/io/trino/cost/TestFilterStatsRule.java

```diff
@@ -25,35 +25,43 @@
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
+import static io.trino.SystemSessionProperties.DEFAULT_FILTER_FACTOR_ENABLED;
 import static io.trino.spi.type.DoubleType.DOUBLE;
 import static io.trino.sql.planner.iterative.rule.test.PlanBuilder.expression;
 import static io.trino.testing.TestingSession.testSessionBuilder;
 
 public class TestFilterStatsRule
```

Comment:
I would just split it into two classes. They don't seem to share much now.

Reply:
It's easier to see the impact of filter enabled/disabled side by side, going to keep it as is